### PR TITLE
Fix selecting puppet environment

### LINKED ISSUE #7: Default puppetclass_ids do not work
Expected behaviour:
When `default_values.puppetclass_ids` is defined and the user selects a Puppetserver and environment, the defined Puppet classes get automatically assigned to the new host.

Actual behaviour:
The classes are not selected and the user needs to select them manually.

### PR DESCRIPTION
**Pull request by kamils-iRonin**
Fixes #7

### CODE REVIEW

**Comment by kamils-iRonin** [comment]
the settings file needs to be updated when this is merged (rename `puppet_env` to `puppet_env_id`)

**Comment by timogoebel** [comment]
@kamils-iRonin: Thanks, is it feasible to add a test that catches these issues?

File: src/containers/NewHostContainer/__tests__/index_test.js [hunk 1]
```diff
@@ -38,7 +38,17 @@ test('Mounted NewHostContainer', async () => {
...
+  wrapper.update()
+
+  const puppetEnvIdSelect = wrapper.find('Select[placeholder="hosts_form.placeholders.puppet_env_id"]')
+  puppetEnvIdSelect.instance().selectValue({ value: 'MDE6RW52aXJvbm1lbnQtMA==', label: 'env2' })
+
+  await wait()
+  wrapper.update()
+
+  const puppetclassIdsSelect = wrapper.find('Select[placeholder="hosts_form.placeholders.puppetclass_ids"]')
+  puppetclassIdsSelect.instance().selectValue({ value: 'MDE6UHVwcGV0Y2xhc3MtMg==', label: 'ppclass2' })
```
Comment (kamils-iRonin) [inline src/containers/NewHostContainer/__tests__/index_test.js]: `environmentId` and `puppetclassIds` are now selected "manually" (other than the default values)

**Comment by laugmanuel** [comment]
Successfully tested the fix and it fixes Issue #7 for me.

**Comment by laugmanuel** [comment]
Thanks @kamils-iRonin !

**Pull request merged**